package.json

```diff
@@ -40,15 +43,19 @@
     "eslint-plugin-eslint-comments": "^3.2.0",
     "eslint-plugin-import": "^2.24.1",
     "eslint-plugin-jsdoc": "^37.1.0",
-    "eslint-plugin-mocha": "^10.0.3",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-tsdoc": "^0.2.14",
     "eventemitter2": "^6.0.0",
+    "expect": "^29.3.1",
     "https-browserify": "^1.0.0",
-    "jest": "^26.0.1",
+    "jest": "^29.3.1",
+    "jest-mock": "^29.3.1",
+    "karma": "^6.4.1",
+    "karma-chrome-launcher": "^3.1.1",
+    "karma-jasmine": "^5.1.0",
+    "karma-webpack": "^5.0.0",
```

Comment:
Are these needed at the top level? I thought they were only used for xrpl.js integration tests.

Reply:
Fixed